tabo-hdinsight/Azuredeploy.json

```diff
@@ -0,0 +1,555 @@
+    "TaboLocation": "[resourceGroup().location]",
+    "TaboVirtualMachineName": "[concat('v', uniquestring(resourcegroup().id))]",
+    "TaboVirtualmachinesize": "standard_A3",
+    "TaboAdminUsername": "azureuser",
+    "TaboAdminPassword": "Password@123",
+    "TaboVirtualNetworkName": "[concat('vn', uniquestring(resourcegroup().id))]",
+    "TaboStorageAccountName": "[concat('stor', uniquestring(resourcegroup().id))]",
+    "TaboStorageAccountType": "Standard_LRS",
+    "TaboAddressPrefix": "10.2.0.0/24",
+    "TaboSubnetName": "[concat('sub', uniquestring(resourcegroup().id))]",
+    "TaboSubnetPrefix": "10.2.0.0/24",
+    "TaboPublicIpAddressName": "[concat('pip', uniquestring(resourcegroup().id))]",
+    "TaboPublicIpAddressType": "static",
+    "TaboSqlConnectivityType": "Public",
+    "TaboSqlPortNumber": "1433",
+    "TaboSqlStorageDisksCount": 1,
+    "TaboSqlStorageWorkloadType": "General",
+    "TaboSqlStorageDisksConfigurationType": "RAID0",
+    "TaboSqlStorageStartingDeviceId": 1,
+    "TaboSqlAutopatchingDayOfWeek": "Sunday",
+    "TaboSqlAutopatchingStartHour": "02:00",
+    "TaboSqlAutopatchingWindowDuration": "60",
+    "TaboSqlAuthenticationLogin": "azureuser",
+    "TabooSqlAuthenticationPassword": "Password@123",
+    "TaboNetworksecuritygroupname": "[concat('nsg', uniquestring(resourcegroup().id))]",
+    "TaboNetworkinterfacename": "[concat('nic', uniquestring(resourcegroup().id))]",
+    "TaboVmdns": "[concat('tab', uniquestring(resourcegroup().id))]",
+    "TaboScriptfolder": ".",
+    "TaboScriptfilename": "sqlnew.ps1",
+    "TaboServiceBusNamespaceName": "[concat('sb', uniquestring(resourcegroup().id))]",
+    "TaboEhVersion": "2015-08-01",
+    "TaboDefaultSASKeyName": "RootManageSharedAccessKey",
+    "TaboDefaultAuthRuleResourceId": "[resourceId('Microsoft.ServiceBus/namespaces/authorizationRules', variables('TaboServiceBusNamespaceName'), variables('TaboDefaultSASKeyName'))]",
+    "TaboEventhubin": "IotInputxx",
+    "TaboEventhubout": "Iotoutputxx",
+    "TaboInKeyP": "DG4TcCs+XnDN83VZcT2e5NfVeSFvTuiYUiWtXhxnJ2s=",
+    "TaboInKeyS": "IAvcmGyxkAHoOvVRLrbuPRgGw+RqAa/KedmOaQYxTt8=",
+    "TaboOutKeyP": "Xo5cCWHgJh3Sag00I5BiG6PlfTHBuSsCWQOyj8wFdhU=",
+    "TaboOutKeyS": "ANYLcW/XZBhuuXfzaqwyUfxIa1HngNTGvgMyyjvnpj4=",
+    "taboServicebusSku": "standard",
+    "taboServiceBusApiVersion": "2015-08-01",
+    "TaboInpolicy": "iotinput",
+    "TaboOutpolicy": "iotoutput",
+    "TaboStorageAccountNamex": "[concat('sto',uniquestring(resourceGroup().id))]",
+    "TaboDefaultApiVersion": "2015-05-01-preview",
+    "TaboClusterApiVersion": "2015-03-01-preview",
+    "TaboClustertype": "Spark",
+    "TaboClustername": "[concat('hd', uniquestring(resourcegroup().id))]",
+    "TaboClusterLoginUserName": "admin",
+    "TaboClusterLoginPassword": "Password@123",
+    "TaboSshUserName": "azureuser",
+    "TaboSshPassword": "Password@123",
```

Comment:
definitely shouldn't hard-code passwords and keys in this template; please make these parameters